filter-effects-2/Overview.bs

```diff
@@ -10,12 +10,15 @@ Link Defaults: css-transforms-1 (property) transform, svg (property) color-inter
 Group: fxtf
 Editor: Dean Jackson, Apple Inc. http://www.apple.com/, dino@apple.com, w3cid 42080
 Former Editor: Dirk Schulze, Adobe Inc., dschulze@adobe.com, w3cid 51803
+!Author: Mason Freed, Google, http://www.google.com/, masonfreed@google.com
```

Comment:
Do other specs have this field?

Reply:
I don't know. I read the description of "Author" [here](https://www.w3.org/Signature/Contributor.html), and I think I fit that description. I wrote 90% of this text. Let me know what you think I should do here.

Reply:
We've never used that sort of field in the metadata (or Contributor, etc). Typically, you're either an Editor, or you're thanked in the Acknowledgements section.

Reply:
Ok. I found that issue referred to [here](https://github.com/tabatkins/bikeshed/issues/885). 
I've removed my name and added an acknowledgements section. I added everyone from [the github issue](https://github.com/w3c/fxtf-drafts/issues/53). I also added Chris as an editor - I think that's correct, right?